tests/acceptance/cli/test_federated_cli_lifecycle.py

```diff
@@ -0,0 +1,38 @@
+"""
+ This file is part of nucypher.
+
+ nucypher is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+import pytest_twisted as pt
+
+from tests.acceptance.cli.lifecycle import run_entire_cli_lifecycle
+
+
+@pt.inlineCallbacks
+def test_federated_cli_lifecycle(click_runner,
```

Comment:
Any reason why we need the `testerchain` fixture for the federated tests? Can we pass `testerchain = None` to the `run_entire_cli_lifecycle` method, or is there something precluding us from doing so?

Reply:
Good eye @derekpierre !